src/models/accountModel.ts

```diff
@@ -0,0 +1,58 @@
+import dbConnection from "../dbConnection/dbConnection";
+
+export interface Account {
```

Comment:
It would be nice to call this `IAccount` to make it clearer that it's an interface